examples/backend/main.tf

```diff
@@ -0,0 +1,24 @@
+ terraform {
+  # https://www.terraform.io/docs/backends/types/s3.html
+  backend "s3" {
+      region         = "<region>"
+      bucket         = "<s3-bucket-name>"
+      key            = "terraform.tfstate"
+      dynamodb_table = "terraform-state-lock"
+      encrypt        = true
+    }
+ }
+
+module "eks-jx" {
+  source  = "jenkins-x/eks-jx/aws"
+}
+
+output "vault_user_id" {
+  value       = module.eks-jx.vault_user_id
+  description = "The Vault IAM user id if one got created"
+}
+
+output "vault_user_secret" {
+  value       = module.eks-jx.vault_user_secret
+  description = "The Vault IAM user secret if one got created"
+}
```

Comment:
Missing new line. Also shouldn't outputs be in their own `output.tf` file?

Reply:
They don't have to be. For such a small example having everything in one file might be easier to take in. Not sure. I can change.